### PR TITLE
Hotfix/pool field missing and Action publish command fix 

### PR DESCRIPTION
**Pull request by Caitlyn-Mills**
Summary of changes: 

Add 'Return' value to expected values in Allocation_status__c picklist and check that the picklist length is 5

Remove --dry-run tag from publish command

### CODE REVIEW

File: .github/workflows/review-smoke-test.yml [hunk 1]
```diff
@@ -24,6 +24,9 @@ jobs:
         - name: Install SFDX
           run: npm install -g  sfdx-cli
           
+        - name: Run NPM Install
+          run: npm install
```
Comment (azlam-abdulsalam) [inline .github/workflows/review-smoke-test.yml:28]: Can you make this as part of a seperate PR?

File: src/utils/scratchOrgUtils.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default class ScratchOrgUtils {
                                 this.sfdxAuthUrlFieldExists = true;
                             }
 
-                            if (field.name === 'Allocation_status__c' && field.picklistValues.length === 4) {
+                            if (field.name === 'Allocation_status__c' && field.picklistValues.length === 5) {
```
Comment (azlam-abdulsalam) [inline src/utils/scratchOrgUtils.ts]: This will break backward compatibility, Probably check greater than 4?

Comment (azlam-abdulsalam) [reply src/utils/scratchOrgUtils.ts]: Or may be as this functionality is not required for sfpowerkit, the ideal fix would be is to work on the warning

**Review by azlam-abdulsalam — CHANGES_REQUESTED** [review]
Please have a look at the comments

File: .github/workflows/review-smoke-test.yml (hunk 1 above)
Comment (Caitlyn-Mills) [reply .github/workflows/review-smoke-test.yml:28]: I have done this already in main - not sure why it's showing in this PR exactly. I pulled into this branch from main, but there aren't changes.

**Pull request closed without merging**